smokes/e2e/pages/waterfall.ts

```diff
@@ -5,12 +5,13 @@ import {browser, by, element, ExpectedConditions as EC} from 'protractor';
 import { BasePage } from "./base";
 
 export class WaterfallPage extends BasePage {
-    constructor(builder) {
-        {
-          super();
-          let thisFn = (() => { return this; }).toString();
-          let thisName = thisFn.slice(thisFn.indexOf('return') + 6 + 1, thisFn.indexOf(';')).trim();
-        }
+    builder: string;
+
+    constructor(builder: string) {
+        super();
+        //  let thisFn = (() => { return this; }).toString();
```

Comment:
why keep commented out code?

Reply:
Fixed by removing the commented out code